pom.xml

```diff
@@ -216,6 +222,12 @@
 		</dependency>
 
 		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
```

Comment:
Yes, it is a transitive dependency from aws-java-sdk, without it's not working :/